react/SummaryTotalizers.tsx

```diff
@@ -54,6 +71,14 @@ function SummaryTotalizers({
       )}
     </Fragment>
   )
+  }
+
+  return null
+}
+
+interface SummaryTotalizersProps {
+  showTotal: boolean
+  showDeliveryTotal: boolean
```

Comment:
`Props` are already defined in the beginning of this file, there is no need to add this interface right here. Could you remove it?